packages/services/hmi-server/src/main/java/software/uncharted/terarium/hmiserver/proxies/xdd/DocumentProxy.java

```diff
@@ -0,0 +1,21 @@
+package software.uncharted.terarium.hmiserver.proxies.xdd;
+
+import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
+
+import software.uncharted.terarium.hmiserver.models.xdd.XDDArticlesResponseOK;
+import software.uncharted.terarium.hmiserver.models.xdd.XDDResponse;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@RegisterRestClient(configKey = "xdd-document-service")
+@Produces(MediaType.APPLICATION_JSON)
+public interface DocumentProxy {
+	@GET
+	@Path("articles")
+	XDDResponse<XDDArticlesResponseOK> getDocuments(
```

Comment:
Just have the proxy return a `Response`

Reply:
By returning just the response, we gain the benefit of letting the client or the front-end takes care of extracting the fields and handling the response whether it is a successful one or not. However, in this case I don't see a value for having all XDD models at the backend since they are not referenced anymore!

Reply:
eventually, it would be nice to do some checking on the backend. I thought it best for the dec 8 demo to have simple wiring and we can expand later

Reply:
Note that returning a Response does break the consistency. This is because XDD returns an object which now the front-end would have to handle differently from the data returned by the data-service. In the case of fetching "models" for instance from the data-service, the proxy os returning a list of models as opposed to a Response object. Consequently, the code at the front-end would sometimes call an endpoint and expects to get a list while in other cases would have to extract the list from a response object.

Reply:
As noted earlier, returning a raw response from XDD while ignoring the proper mapping by the proxy has bad side effects that the front-end code is forced to handle, reducing code readability and increasing code duplication because of the need to do re-mapping at the front-end. I've made the change as suggested, but we should re-visit this one later as time permits!

Reply:
done